content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,255 @@
+services product. To do this, go to :menuselection:`Sales app --> Orders --> Quotations`. Then,
+click :guilabel:`Create` to reveal a blank quotation form.
+
+Then, fill out the quotation form with the customer information.
+
+Go to the :guilabel:`Order Lines` tab of the quotation and click :guilabel:`Add a Product`. Then,
+select the *prepaid services product* configured in the steps above. Update the :guilabel:`Quantity`
+field with the number of hours.
+
+After updating any other necessary information, :guilabel:`Confirm` the quotation. This converts the
+quotation into an :abbr:`SO (sales order)`.
+
+Create and send an invoice for prepaid services
+-----------------------------------------------
+
+Once the :abbr:`SO (sales order)` has been confirmed, click the :guilabel:`Create Invoice` button.
+This opens a :guilabel:`Create invoices` pop-up window.
+
+If no down payment is collected, the :guilabel:`Create Invoice` type can remain as
+:guilabel:`Regular Invoice`. If a :doc:`down payment <../../../sales/sales/invoicing/down_payment>`
+is collected, choose between either :guilabel:`Down payment (percentage)` or :guilabel:`Down payment
+(fixed amount)`.
+
+When the necessary information has been entered, click :guilabel:`Create and View Invoice` or
+:guilabel:`Create Invoice`.
+
+.. tip::
+   Invoices are created in draft mode, so they can be reviewed and edited if necessary.
+
+The invoice can then be :doc:`sent to the customer <../../../finance/accounting/customer_invoices>`
+for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+To create a *Helpdesk* ticket for prepaid services, navigate to :menuselection:`Helpdesk` and click
+the :guilabel:`Tickets` button to reveal a specific team's pipeline. Click :guilabel:`Create` to
+create a new ticket.
+
+On the blank ticket form, create a ticket :guilabel:`Title`, and enter the :guilabel:`Customer`
+information.
+
+When the customer name is added, the :guilabel:`Sales Order Item` field automatically populates with
+the most recent prepaid sales order item that has time remaining.
+
+If a customer has more than one sales order item with remaining time, click the :guilabel:`Sales
+Order Item` field and select the correct item from the drop-down list.
+
+After entering all of the necessary information, click :guilabel:`Save`.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Time spent working on a *Helpdesk* ticket is tracked on the *Timesheets* tab on the specific ticket.
+
+On the ticket detail form, click on the :guilabel:`Timesheets` tab and click :guilabel:`Add a line`.
+Choose an :guilabel:`Employee`, add a :guilabel:`Description` of the task, and enter the number of
+:guilabel:`Hours Spent` working on the task.
+
+As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field,
+at the bottom-right of the tab, is automatically updated.
```

Comment:
In my runbot, I was able to select a timesheet that did notdisplay the remining time. i think it's because there wasn't time left- probably not common, but maybe reference that? I oly disocverd it because I couldn't see the 'Remaining Hours' so I switche the timesheet. I think this is in all runbots if you want to test it. If it's importnat info- could be put into a note.